src/site/_includes/components/Aside.js

```diff
@@ -22,6 +25,33 @@ function Aside(content, type = 'note') {
   const locale = getLocaleFromPath(this.page && this.page.filePathStem);
 
   let prefix;
+
+  // Renders the new aside
+  if (isDesignSystemContext(this.page.filePathStem)) {
+    switch (type) {
```

Comment:
Some quick comments about why there's special-cases here would be good. e.g., "note" will end up with `title` being `undefined`.

Reply:
This is getting deleted out but yep: noted (pun intended)